.travis.yml

```diff
@@ -8,6 +8,7 @@ language: go
 
 go:
   - 1.9.x
+  - 1.10.x
```

Comment:
According to https://github.com/travis-ci/gimme/issues/132, this line needs to be "1.10".

Reply:
Ah, yes, I read about the `1.10` issue, but thought that `1.10.x` would not result in that problem. Apparently though;

> Travis is now recognizing 1.9.x and passing only 1.9 to gimme.

So I wonder if the `1.9.x` should also be updated to be `1.9`

Reply:
IIUC 1.10.x will be valid after 1.10.1 gets released...😵


1.9.x has been valid since 1.9.1.

Reply:
Looks like someone opened a PR to add support for `.x` directly in "gimme"; https://github.com/travis-ci/gimme/pull/130